listenbrainz/spark/spark_dataset.py

```diff
@@ -0,0 +1,206 @@
+import abc
+import time
+from abc import ABC
+from urllib.error import HTTPError
+
+from flask import current_app
+from psycopg2.extras import execute_values
+from psycopg2.sql import Identifier, SQL, Literal
+
+from listenbrainz.db import couchdb, timescale
+
+
+class SparkDataset(ABC):
```

Comment:
docstring

Reply:
Given this is a base class, all the methods needs docstrings.

Reply:
Fixed.